### PR TITLE
Change to 'migrate existing site' docs to reflect the differences I found when following the guide.

### PR DESCRIPTION
**Pull request by lewisheaton404**
## Description

I have made changes to the 'migrate existing site' docs as when I was following the guide recently, what the existing guide suggests did not reflect what I was seeing. I changed the single file as well as updated an image.

Note: In the file, I have removed the line 'Run `Schema Deployment From Data Files` to check that the UDA files have been generated correctly', I don't believe this exists anymore.

## Type of suggestion

* [ ] Typo/grammar fix
* [X] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

I have been using the latest version of Umbraco 13.0.2 at the time.

Final note, this is my first PR in this manor, so go easy please 😆

### CODE REVIEW

**Comment by alina-tincas** [comment]
Hi @lewissmith606 thank you for the PR 🙌 

We will take a look at it as soon as we can 😊

File: umbraco-cloud/getting-started/migrate-existing-site/README.md [hunk 1]
```diff
@@ -196,15 +196,16 @@ In this next part, it is time to generate the so-called UDA-files for all your p
 
 For more details about UDA files, read the [UDA Files](../../set-up/power-tools/generating-uda-files.md#what-are-uda-files) article.
 
+Your local Cloud project will need [Umbraco Deploy](https://www.nuget.org/packages/Umbraco.Deploy.OnPrem) installed, you can check your .csproj file to see if this is already installed. If not installed, install this and check your site is working as expected.
```
Comment (jonat123) [inline umbraco-cloud/getting-started/migrate-existing-site/README.md:199]:
```suggestion

```

Comment (jonat123) [reply umbraco-cloud/getting-started/migrate-existing-site/README.md:199]: @lewissmith606 All Umbraco projects come with Deploy out-of-the-box, so there is no need to check whether Deploy is installed 😊

File: umbraco-cloud/getting-started/migrate-existing-site/README.md [hunk 2]
```diff
@@ -196,15 +196,16 @@ In this next part, it is time to generate the so-called UDA-files for all your p
...
 * Make sure the folder `/Umbraco/Deploy/Revision` on your Umbraco Cloud project is empty
   * If you have any files in the folder, you can safely remove those at this point
 * Start your local cloud project
 * Go to the settings section in your project
-* Select `Extract Schema To Data Files` Command from the **Deploy Operations** drop-down and click **Trigger Operation**.
+* Go to the Deploy tab
+* Under **Deploy Operations** > **Export Schema To Data Files** click **Export Schema**.
   * Generating the UDA files may take a while, depending on the extent of your project
-  * You will see a `Last deployment operation completed` status once the export is done
-  * Run `Schema Deployment From Data Files` to check that the UDA files have been generated correctly
```
Comment (jonat123) [inline umbraco-cloud/getting-started/migrate-existing-site/README.md:206]: I think this should still be a part of the guide, the only thing is to rename it to: `Run `Update Umbraco Schema` to check that the UDA files have been generated correctly` as it is to just make sure that everything is generated correctly 😊

File: umbraco-cloud/getting-started/migrate-existing-site/README.md [hunk 3]
```diff
@@ -196,15 +196,16 @@ In this next part, it is time to generate the so-called UDA-files for all your p
...
-  * When you see a `Last deployment operation completed` status, it means everything is working as expected
+  * When you see a `Last deployment operation completed` status at the top of this page, it means everything is working as expected
 * You should now see that your `/Umbraco/Deploy/revision` folder has been populated with UDA files corresponding to your project's metadata
 
 ![Run echo > deploy-export](images/deployDashBoard.png)
```
Comment (jonat123) [inline umbraco-cloud/getting-started/migrate-existing-site/README.md:211]: If you add the above change, Could you maybe highlight both buttons, so the one for `update Umbraco Schema` and` export schema `? 😊

**Review by jonat123 — CHANGES_REQUESTED** [review]
@lewissmith606 Thanks for your PR! 😄

I have made a couple of comments and suggestions 😊

Please let me know what you think 😄

File: umbraco-cloud/getting-started/migrate-existing-site/README.md (hunk 1 above)
Comment (lewissmith606) [reply umbraco-cloud/getting-started/migrate-existing-site/README.md:199]: Great, I’ll go ahead and remove this line @jonat123

File: umbraco-cloud/getting-started/migrate-existing-site/README.md (hunk 3 above)
Comment (lewisheaton404) [reply umbraco-cloud/getting-started/migrate-existing-site/README.md:211]: @jonat123 I have made that change, could you check I've got the right of the stick here :)

File: umbraco-cloud/getting-started/migrate-existing-site/README.md (hunk 2 above)
Comment (lewisheaton404) [reply umbraco-cloud/getting-started/migrate-existing-site/README.md:206]: Have made that change @jonat123

File: umbraco-cloud/getting-started/migrate-existing-site/README.md (hunk 3 above)
Comment (jonat123) [reply umbraco-cloud/getting-started/migrate-existing-site/README.md:211]: Cool!

I think I was a bit unclear here 🙈 What I was thinking was that in the image, it should highlight both buttons and not only the export schema button 😊

Comment (lewisheaton404) [reply umbraco-cloud/getting-started/migrate-existing-site/README.md:211]: Ah, got you, okay that will take me a bit of time as ill have to get the code up and running again, one for next week @jonat123

**Review by jonat123 — APPROVED** [review]
@lewisheaton404 Thanks for looking into my suggestions :D 

It looks good to me, so i will merge it in :-)

**Pull request merged**